scripts/get_redash_api_key.sh

```diff
@@ -18,7 +18,7 @@ else
   helm install "forms-flow-analytics" ./forms-flow-analytics-chart -n "$NAMESPACE"
 fi
 
-REDASH_URL="https://$ANALYTICS_SUBDOMAIN-$NAMESPACE.$DOMAIN_NAME"
+REDASH_URL="http://$ANALYTICS_SUBDOMAIN-$NAMESPACE.$DOMAIN_NAME"
```

Comment:
Why is it changed to http ?

Reply:
we were getting ssl certificate issue while installing. So Amal has suggested to change to http.

Reply:
if you are using http, I would suggest using service instead of ingress.

Reply:
@sumesh-aot  we can not use service name while running the script the service only can be accessible within the same network.